js/viewer/annotation/base.js

```diff
@@ -6,16 +6,25 @@ function Base(attrs){
     this.isDrawing = false;
     this.constants = attrs.parent.parent.parent.all_config.constants;
     this.urlParams = attrs.parent.parent.parent.parameters;
+    this.annotationUtils = new AnnotationUtils();
+
+    // this is used to make sure that any property that is added to the SVG for display purpose, is not added while saving
+    this.addedProperties = {
```

Comment:
In pure Javascript, there's no concept of private/public APIs. So instead we use a naming convention for this. If an attribute starts with `_`, it's private, otherwise, it will be considered public. So changed `addedProperties` and `ignoredAttributes` to `_addedProperties` and `_ignoredAttributes`.

Reply:
Done